modules/ROOT/pages/afm-view-test-history.adoc

```diff
@@ -1,29 +1,22 @@
-= View Test History
-ifndef::env-site,env-github[]
-include::_attributes.adoc[]
-endif::[]
-
-:imagesdir: ../assets/images
+= Viewing Test History
```

Comment:
Viewing Test Histories?

Reply:
Hmm, I think History can be considered plural

Reply:
I could say "Viewing the History of Tests", but that seems overkill